README.md

```diff
@@ -37,6 +37,51 @@ https://annu12340.github.io/Javascript-Notes/
 - JavaScript ES2020
 - Tricky JavaScript interview questions and answers
 
+##Infinities
+
+ Infinity is a property of the global object , or in other words, a variable in global scope. 
+ The initial value of Infinity is Number.POSITIVE_INFINITY . The JavaScript keyword Infinity 
```

Comment:
Hey  Infinity is a special value in JavaScript (I don't think it is a property)

Check out this site https://medium.com/swlh/what-is-infinity-in-javascript-%EF%B8%8F-1faf82f100bc

Reply:
I think it's a property. Many conflict on this on web. Found this:- 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity

Reply:
Done the requested changes. Please review

Reply:
> I think it's a property. Many conflict on this on web. Found this:-
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity

Oooh
okay then 👍